src/patterns/equality-information/index.md.njk

```diff
@@ -0,0 +1,166 @@
+---
+title: Equality information
+description: This pattern explains how to ask users for equality information
+section: Patterns
+theme: Ask users for…
+backlog_issue_id: 180
```

Comment:
Do we need to rename the corresponding backlog issue? Currently called 'Protected characteristics'.

Reply:
Probably done as part of https://github.com/alphagov/govuk-design-system/issues/1545, I'll add it to that card